internal/services/notificationhub/notification_hub_namespace_data_source.go

```diff
@@ -54,6 +54,11 @@ func dataSourceNotificationHubNamespace() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"zone_redundant": {
```

Comment:
Could you rename this to `zone_redundancy_enabled` to follow our naming convention of `something_enabled` for booleans? ([ref](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-naming.md))

```suggestion
			"zone_redundancy_enabled": {
```